app/posts/screening-invite/2025/09/2025-08-13-invite-breast-screening-pilot-user-research.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Researching the impact of piloting digital-first invitations in breast screening
+description: Content and interaction design work for the 2024 breast screening invitations Pilot.
+date: 2025-09-04
+---
+Team Invite is working on inviting people to breast screening appointments using a digital-first communication strategy.  
+In our first post, [we gave a high-level overview of the pilot work](https://design-history.prevention-services.nhs.uk/screening-invite/2025/06/invite-pilot-overview/). In the second one, [we deep dived into content and interaction design iterations](https://design-history.prevention-services.nhs.uk/screening-invite/2025/08/invite-breast-screening-pilot-design-iterations/). In this one we have summarised what we found out during the pilot phase about user experience and operational impact. 
+
+
+## What we wanted to find out in the pilot
+
+At a high level we were keen to understand why messages in different channels might be read and whether they impact attendance and overall experience.
+
+Some of the details we wanted to learn included:
+*	would not sending a map have an impact?
+*	would participants go back to their invite on their phone?
+*	would participants see the guidance link in the digital message?
+*	how might breast screening offices be impacted?
+
+## The user research methods we used
+
+It’s hard to catch that moment when someone receives a message. You can’t observe someone indefinitely and we didn’t have the data agreements in place to call people.
+
+Instead, we started with some content and usability testing of the NHS App invitation to try and get some sense of how they’d respond to it.
+
+Then for the pilot itself we did at least have the data that told us who received which message and when.
+
+Using this and then some significant help from the breast screening offices themselves, we managed to intercept the participants that we sent messages to, either at the end of their appointment or on the phone if they called. Breast screening offices also called anyone who didn’t attend their appointment.
```

Comment:
I wonder if this could be bullets of what you did, that follows-on from the line above.

Instead we began by testing NHS App invitations in moderated sessions, to see how people responded.

During the pilot we:
- tracked who got which message and when
- asked screening staff helped us speak to people – either after their appointments or when they called in
- asked screening staff to call people who didn't attend